packages/iTwinUI-react/src/core/Wizard/Stepper.test.tsx

```diff
@@ -0,0 +1,220 @@
+  const stepper = (
+    <Stepper
+      currentStep={42}
+      steps={[
+        {
+          name: 'Step One',
+        },
+        {
+          name: 'Step Two',
+        },
+        {
+          name: 'Step Three',
+        },
+      ]}
+    />
+  );
+
+  const { getByText } = render(stepper);
+  const thirdStep = getByText('Step Three') as HTMLSpanElement;
+  expect(thirdStep.parentElement?.className).toContain('iui-current');
+});
+
+it('should only display summary of current step name in long stepper', () => {
+  const stepper = (
+    <Stepper
+      currentStep={2}
+      type='long'
+      steps={[
+        {
+          name: 'Step One',
+        },
+        {
+          name: 'Step Two',
+        },
+        {
+          name: 'Step Three',
+        },
+      ]}
+    />
+  );
+
+  const { getByText, queryByText } = render(stepper);
+
+  expect(getByText('Step 3 of 3:')).toHaveClass(
+    'iui-stepper-steps-label-count',
+  );
+  expect(getByText('Step Three')).toHaveClass('iui-stepper-steps-label');
+
+  getByText('1');
+  getByText('2');
+  getByText('3');
+
+  expect(queryByText('Step One')).toBeNull();
+  expect(queryByText('Step Two')).toBeNull();
+});
+
+it('should display localized string in long stepper', () => {
+  const stepper = (
+    <Stepper
+      currentStep={1}
+      type='long'
+      localization={{
+        stepsCountLabel: (currentStep, totalSteps) =>
+          `This is ${currentStep} of great ${totalSteps}:`,
+      }}
+      steps={[
+        {
+          name: 'Step One',
+        },
+        {
+          name: 'Step Two',
+        },
+        {
+          name: 'Step Three',
+        },
+      ]}
+    />
+  );
+
+  const { getByText } = render(stepper);
+
+  getByText('This is 2 of great 3:');
+  getByText('Step Two');
+});
+
+it('should display tooltip upon hovering step if description provided', async () => {
+  jest.useFakeTimers();
+
+  const stepper = (
+    <Stepper
+      currentStep={1}
+      steps={[
+        {
+          name: 'Step One',
+          description: 'Step one tooltip',
+        },
+        {
+          name: 'Step Two',
+          description: 'Step two tooltip',
+        },
+        {
+          name: 'Step Three',
+        },
+      ]}
+    />
+  );
+
+  render(stepper);
+
+  expect(document.querySelector('.iui-tooltip')).toBeNull();
+  expect(screen.queryByText('Step one tooltip')).toBeNull();
+  fireEvent.mouseEnter(screen.getByText('Step One'), { bubbles: true });
+  const tooltip = document.querySelector('.iui-tooltip') as HTMLElement;
+  expect(tooltip).toBeVisible();
+  expect(tooltip).toHaveTextContent('Step one tooltip');
+
+  fireEvent.mouseLeave(screen.getByText('Step One'), { bubbles: true });
+  await waitForElementToBeRemoved(tooltip);
+
+  fireEvent.mouseEnter(screen.getByText('Step Three'), { bubbles: true });
+  expect(screen.queryByText('Step three tooltip')).toBeNull();
```

Comment:
Same thing I said for WorkflowDiagram.test.tsx

Reply:
Test changed